src/components/application_manager/rpc_plugins/vehicle_info_plugin/src/commands/mobile/on_vehicle_data_notification.cc

```diff
@@ -68,48 +68,48 @@ void OnVehicleDataNotification::Run() {
   std::vector<smart_objects::SmartObject> appSO;
 
   const VehicleData& vehicle_data = MessageHelper::vehicle_data();
+  const smart_objects::SmartObject& incoming_msg_params =
+      (*message_)[strings::msg_params];
   VehicleData::const_iterator it = vehicle_data.begin();
 
   for (; vehicle_data.end() != it; ++it) {
-    if (true == (*message_)[strings::msg_params].keyExists(it->first)) {
-      LOG4CXX_ERROR(logger_, "vehicle_data nanme" << it->first);
-      auto vehicle_data_value =
-          (*message_)[strings::msg_params][it->first].asInt();
-
-      application_manager_.IviInfoUpdated(it->second, vehicle_data_value);
-
+    // LOG4CXX_ERROR(logger_, "Element nanme" << it->first);
```

Comment:
@ValeriiMalkov please remove commented code

Reply:
@AKalinich-Luxoft That was fixed and force updated